src/Parser/Anime/AnimeParser.php

```diff
@@ -90,10 +90,10 @@ public function getImageURL(): string
     public function getSynopsis(): ?string
     {
         $synopsis = JString::cleanse(
-            $this->crawler->filterXPath('//meta[@property=\'og:description\']')->attr('content')
+            $this->crawler->filterXPath('//p[@itemprop=\'description\']')->html()
```

Comment:
is there any chance that instead of `p` there would be a different tag?

Reply:
For anime no. The `p` tag is there for shows that have a synopsis ([example](https://myanimelist.net/anime/28977/Gintama%C2%B0)) as well as if there is no synopsis ([example](https://myanimelist.net/anime/24857/Akagaki_Genzou__Tokuri_no_Wakare)). 

Could also change it to just `[@itemprop=\'description\']` to make sure it keeps matching even if the MAL developpers decide to change the tag to something else (like `span` at manga).